tests/test_webservice.py

```diff
@@ -215,16 +209,18 @@ def test_200_on_request_with_nones(self):
         response = self.response
         self.assertEqual(0.01, model.risk_score)
 
-    @httprettified
     def test_200_with_email_hashing(self):
-        uri = "/".join([self.base_uri, self.type])
+        uri = "/".join(["/minfraud/v2.0", self.type])
 
-        httpretty.register_uri(
-            httpretty.POST,
-            uri=uri,
-            status=200,
-            body=self.response,
-            content_type=f"application/vnd.maxmind.com-minfraud-{self.type}+json; charset=UTF-8; version=2.0",
+        last = None
+
+        def custom_handler(r):
+            nonlocal last
+            last = r
+            return '{"status": 204}'
```

Comment:
The library seems to have [a better way](https://pytest-httpserver.readthedocs.io/en/latest/howto.html#querying-the-log) for querying the request data but it is not yet available in the latest release.